orcestra_book/_ext/reports.py

```diff
@@ -33,7 +33,7 @@ def load_frontmatter(path, derive_flight=False):
 @lru_cache
 def create_badge(src, cat_id):
     """Return an HTML node based on a category id."""
-    with open(src / "flight_reports" / "categories.yaml", "r") as fp:
+    with open(src / "reports" / "categories.yaml", "r") as fp:
```

Comment:
These categories are as far as I understand, specific to HALO. We might want to reflect this in the function name, or might want to design this facility such that we could handle different categories for different platforms.

Reply:
Yes that's true. I think for now, we should simply rename the function (and the role?) and extend the functionalist when needed